draft-ietf-jose-hpke-encrypt.md

```diff
@@ -198,7 +198,7 @@ In HPKE JWE Integrated Encryption:
 - JWE Initialization Vector and JWE Authentication Tag MUST NOT be present.
 - JWE AAD MAY be present.
 - JWE Ciphertext is ciphertext as defined in Section 5.2 of {{RFC9180}}.
-- The HPKE info parameter MUST be set to an empty string.
+- The HPKE info parameter defaults to the empty string; mutually known private information MAY be used instead. The concept of mutually known private information is defined in {{NIST.SP.800-56Ar3}} as an input to the key derivation function.
```

Comment:
As far as I can tell, the phrase "mutually known private information" appears only once in the 141 page NIST https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-56Ar3.pdf document as part of the definition of a sub-field of a field in recommendation for KDF input that isn't clear that is actually related to HPKE or this document. I don't see how this sentence is accurate or helpful.